.azure-devops/full-pipeline.yml

```diff
@@ -12,3 +12,21 @@ jobs:
       - template: ./steps/devcerts.yml
       - template: ./steps/edgewebview.yml
       - template: ./steps/test.yml
+
+  - job: Mac
+    pool:
+      name: OPX Mac Dogfood Pool
+    steps:
+    - template: ./steps/npm-install.yaml
+    - template: ./steps/build.yaml
+    - template: ./steps/run-tests.yaml
+    - template: ./steps/cleanup.yaml
+
+  - job: Linux
+    pool:
+      name: Azure Pipelines
+      vmImage: ubuntu-latest
```

Comment:
The Windows job doesn't need this `vmImage` argument as well?

Reply:
Nope . . . there is only one image in that pool.